### PR TITLE
Contest page

### LINKED ISSUE #5817: Add contest page to BuyersGuide routes
Criteria:
- [ ] Add contest.html buyersguide 
- [ ] Add /about/contest/ to buyersguide routes 
- [ ] Make sure we copy the existing contest page to the new namespaced location 
- [ ] Add unit test for contest routeable view

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #5817 

Criteria:
- [x] Add contest.html buyersguide 
- [x] Add /about/contest/ to buyersguide routes 
- [x] Make sure we copy the existing contest page to the new namespaced location 
- [x] Add unit test for contest routeable view

### CODE REVIEW

File: network-api/networkapi/wagtailpages/migrations/0023_add_localizable_field_names.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('wagtailpages', '0022_product_voting'),
+        ('wagtailpages', '0019_pni_snippet_conversion'),
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/migrations/0023_add_localizable_field_names.py:10]: Dependency change from merged migrations

File: network-api/networkapi/buyersguide/templates/about/contest.html [hunk 2]
```diff
@@ -0,0 +1,377 @@
+{% extends "../bg_about_without_nav.html" %}
+
+{% comment %}
+  This is legal text and should only be translated by qualified lawyers.
+  For this purpose we're not translating this template.
+{% endcomment %}
```
Comment (KalobTaulien) [inline network-api/networkapi/buyersguide/templates/about/contest.html:6]: Copied the contest.html page to the namespaced dir. Only thing I added was this comment.

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 3]
```diff
@@ -917,6 +917,11 @@ def about_mss_page(self, request):
         self.template = "about/minimum_security.html"
         return self.serve(request)
 
+    @route(r'^about/contest/$', name='contest')
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:920]:
```suggestion
    @route(r'^contest/$', name='contest')
```

**Review by Pomax — CHANGES_REQUESTED** [review]

File: network-api/networkapi/wagtailpages/migrations/0023_add_localizable_field_names.py (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/wagtailpages/migrations/0023_add_localizable_field_names.py:10]: this got updated in https://github.com/mozilla/foundation.mozilla.org/pull/5623/files already so we don't need that here.

File: network-api/networkapi/buyersguide/templates/about/contest.html [hunk 4]
```diff
@@ -0,0 +1,377 @@
+{% extends "../bg_about_without_nav.html" %}
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/about/contest.html:1]: let's move this up one dir

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 5]
```diff
@@ -917,6 +917,11 @@ def about_mss_page(self, request):
         self.template = "about/minimum_security.html"
         return self.serve(request)
 
+    @route(r'^contest/$', name='contest')
+    def about_contest(self, request):
+        self.template = "about/contest.html"
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:922]: let's move this up one dir

**Review by Pomax — CHANGES_REQUESTED** [review]
we'll need the context vars for contest and about pages

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 6]
```diff
@@ -889,33 +889,66 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
...
 
     @route(r'^about/methodology/$', name='methodology-view')
     def about_methodology_page(self, request):
-        self.template = "about/methodology.html"
-        return self.serve(request)
+        context = self.get_context(request)
+        context['pagetype'] = 'about'
+        context['pageTitle'] = pgettext(
+            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+            '*privacy not included') + ' - Methodolofy'
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:932]:
```suggestion
            '*privacy not included') + ' - Methodology'
```

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 7]
```diff
@@ -889,33 +889,66 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
 
     @route(r'^about/$', name='how-to-use-view')
     def about_page(self, request):
-        self.template = "about/how_to_use.html"
-        return self.serve(request)
+        context = self.get_context(request)
+        context['pagetype'] = 'about'
+        context['pageTitle'] = pgettext(
+            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+            '*privacy not included') + ' - Why to Use This Guide'
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:896]:
```suggestion
            '*privacy not included') + ' - How to Use This Guide'
```

File: network-api/networkapi/wagtailpages/pagemodels/products.py (hunk 6 above)
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:932]:
```suggestion
            '*privacy not included') + ' - Methodology'
```

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 8]
```diff
@@ -889,33 +889,66 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
...
 
     @route(r'^about/meets-minimum-security-standards/$', name='min-security-view')
     def about_mss_page(self, request):
-        self.template = "about/minimum_security.html"
-        return self.serve(request)
+        context = self.get_context(request)
+        context['pagetype'] = 'about'
+        context['pageTitle'] = pgettext(
+            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+            '*privacy not included') + ' - Meets Minimum Security Standards'
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:941]:
```suggestion
            '*privacy not included') + ' - Minimum Security Standards'
```

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 9]
```diff
@@ -889,33 +889,66 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
...
+        return render(request, "about/minimum_security.html", context)
+
+    @route(r'^contest/$', name='contest')
+    def about_contest(self, request):
+        context = self.get_context(request)
+        context['pagetype'] = 'contest'
+        context['pageTitle'] = pgettext(
+            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+            '*privacy not included') + ' - Contest Rules'
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:950]:
```suggestion
            '*privacy not included') + ' - Contest Terms and Conditions'
```

**Review by Pomax — CHANGES_REQUESTED** [review]

**Review by Pomax — APPROVED** [review]

**Pull request merged**